### ISSUE TITLE
In High contrast Aquatic/Dusk/Desert and Night sky themes, keyboard focus indicator is not visible on 'Select a product' drop down menu items in windows.

### ISSUE THREAD

**Issue by msftedad** [telerik/blazor-ui#341]
**Test Environment:**
OS: Windows 11 version 25H2 (OS Build 26200.7271)
Browser: Edge (Version 144.0.3719.18)
URL: https://demos.telerik.com/aspnet-ajax/dropdownlist/examples/overview/defaultcs.aspx

**Pre Requisites:**
Setting > Accessibility > Contrast themes > Aquatic/Desert/Dusk/Night Sky

**Repro Steps:**

1. Open the URL: [https://demos. in telerik.com/aspnet-ajax/dropdownlist/examples/overview/defaultcs.aspx](https://demos.telerik.com/aspnet-ajax/dropdownlist/examples/overview/defaultcs.aspx) in edge browser in windows.
2. Press tab key to reach to the 'Select a product' drop down and expand it.
3. Drop down list item and selected list item will appear.
4. Verify whether In High contrast Aquatic/Dusk/Desert and Night sky themes, keyboard focus indicator is visible on drop down menu items and selected list item in windows or not.

**Actual Result:**
In High contrast Aquatic/Dusk/Desert and Night sky themes, keyboard focus indicator is not visible on 'Select a product' drop down menu items in windows.

**Refer Attachment:**

1. In High contrast modes, keyboard focus indicator is not visible on 'Select a product' drop down menu items in windows.png
2. In High contrast modes, keyboard focus indicator is not visible on 'Select a product' drop down menu items in windows.mp4

https://github.com/user-attachments/assets/153b1b21-c39e-40a7-aa84-7472e0b43e55
<img width="1937" height="1037" alt="Image" src="https://github.com/user-attachments/assets/75869388-ba60-4ecb-b8ab-486ef49c8531" />

**Expected Result:**
In High contrast Aquatic/Dusk/Desert and Night sky themes, keyboard focus indicator should be visible properly on 'Select a product' drop down menu items in windows.

**Comment by dimodi** [comment]
This issue concerns Telerik UI for ASP.NET AJAX, while this repo is for sample apps that use Telerik UI for Blazor. You can open a ticket to the AJAX team from your account at telerik.com.
